en/docs/install-and-setup/setup/distributed-deployment/configuring-wso2-identity-server-as-a-key-manager.md

```diff
@@ -143,7 +143,9 @@ Follow the instructions below to set up and configure the databases for the WSO2
 
 ## Step 4 - Configure WSO2 IS with WSO2 API-M
 
-1. Download the [WSO2 IS Connector]({{base_path}}/assets/attachments/administer/wso2is-km-connector-1.0.16.zip).
+1. Download the WSO2 IS Connector based on the API-M version.
+     - [WSO2 IS Connector for GA Relealse]({{base_path}}/assets/attachments/administer/wso2is-km-connector-1.0.16_ga.zip).
```

Comment:
Spelling mistake
"Relealse" should be "Release"

Reply:
Fixed with 11b0ccf3681d4452e788beee35119a7ae618852d